bert_reranker/main.py

```diff
@@ -138,12 +139,23 @@ def init_model(hyper_params, num_workers, output, validation_interval, gpu, no_m
     else:
         ckpt_to_resume = None
         logger.info('will not try to restore previous models because --no-model-restoring')
-    tb_logger = loggers.TensorBoardLogger('experiment_logs')
-    for hparam in list(hyper_params):
-        tb_logger.experiment.add_text(hparam, str(hyper_params[hparam]))
+
```

Comment:
can you add all these new HP to the check_and_log_hp? So that the code will cehck that they are indeed in the config.yaml file?

Reply:
done :)